CHANGELOG.md

```diff
@@ -78,3 +64,4 @@ and
 * SliceViewModel.newSlice deprecated since version 4.16. Use start instead.
 * Slider.labelsVisible deprecated since version 4.15. Use Slider.visibleElements.labels instead.
 * Slider.rangeLabelsVisible deprecated since version 4.15. Use Slider.visibleElements.rangeLabels instead.
+* widget.renderable deprecated since version 4.19. All properties are automatically tracked now and don't need to be decorated with this decorator.
```

Comment:
Do we also need to include the manually added deprecation items?

Reply:
Already done - as the first block under "Deprecation".

Reply:
Ah yes!